test_name 	raising-catching-exception-in-once
test_description Raising and catching exceptions in once routines. The same exception object should be raised the second time a once routine is called.

copy_sub 	Ace $TEST Ace
copy_raw 	a.e $CLUSTER a.e

compile_melted
compile_result 	ok
c_compile_work
execute_work	NONE exec_output
execute_result ok
compare		exec_output output

cleanup_compile
compile_frozen
compile_result 	ok
c_compile_work
execute_work	NONE exec_output1
execute_result 	ok
compare		exec_output1 output

cleanup_compile
compile_final_keep
compile_result 	ok
c_compile_final
execute_final	NONE exec_output2
execute_result ok
compare		exec_output2 output
 
test_end